appsflyer/build.gradle

```diff
@@ -22,10 +22,10 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     // rudder core sdk
-    compileOnly 'com.rudderstack.android.sdk:core:1.+'
+    implementation 'com.rudderstack.android.sdk:core:[1.2.1,)'
 
     // appsflyer dependencies
-    implementation 'com.appsflyer:af-android-sdk:6.+'
+    implementation 'com.appsflyer:af-android-sdk:6.4.3'
```

Comment:
In [iOS](https://github.com/rudderlabs/rudder-integration-appsflyer-ios/pull/7/files#diff-1fc52cc74b5b5fa77bed772259cb70a30445444ef9681ce39d61d53ac2788d5aR29), you are setting 6.4.3 as a minimum version. Here, you are fixing the version.

Reply:
Which one should we follow for external dependency:

1. Specify a specific version (as in iOS)
2. Specify a minimum version (as in Android)

Reply:
Let's fix the version 6.4.3.